src/ontology/mondo-edit.obo

```diff
@@ -586028,6 +586028,16 @@ property_value: http://purl.org/dc/elements/1.1/date "2024-07-03T14:16:10Z" xsd:
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7719" xsd:anyURI
 
+[Term]
+id: MONDO:1040005
+name: PRPF19-related neurodevelopmental disorder
+def: "Any neurodevelopmental disorder in which the cause of the disease is a mutation in the PRPF19 gene. This disorder is characterised by developmental, motor and speech delay, and facial dysmorphisms. Most patients present mild to moderate intellectual disability, autism spectrum disorder or autistic features, and hypotonia. Other phenotypes observed less frequently include seizures, and visual and cardiac abnormalities." [https://orcid.org/0000-0002-0587-4693, https://orcid.org/0000-0002-0987-9862, PMID:37962958]
+is_a: MONDO:0100500 ! Mendelian neurodevelopmental disorder
+intersection_of: MONDO:0100500 {source="https://orcid.org/0000-0002-0587-4693"} ! Mendelian neurodevelopmental disorder
+intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/17896 {source="https://orcid.org/0000-0002-0587-4693"} ! PRPF19
```

Comment:
we currently do not curate sources for logical definitions. I will move this source to the SubClassOf

Reply:
I forget these little details. Thank you!